cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -2634,7 +2639,7 @@ macro(build_bzip2)
                       BUILD_IN_SOURCE 1
                       BUILD_COMMAND ${MAKE} libbz2.a ${MAKE_BUILD_ARGS}
                                     ${BZIP2_EXTRA_ARGS}
-                      INSTALL_COMMAND ${MAKE} install PREFIX=${BZIP2_PREFIX}
+                      INSTALL_COMMAND ${MAKE} install -j1 PREFIX=${BZIP2_PREFIX}
```

Comment:
any reason why this is not `${MAKE_BUILD_ARGS}` and has to be hardcoded to `-j1`?